2.1.0/nanoserver1803/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build stage
+FROM golang:1.12.9-windowsservercore-1809 as tmp
```

Comment:
As discussed, you use this (even though there is a golang 1.12.9 "nanoserver" version) because the servercore as the tools that are needed to properly do the build stage.